src/common/evaluation/QA/eval_depthmap_models/src/utils.py

```diff
@@ -329,10 +329,18 @@ def draw_stunting_diagnosis(df: pd.DataFrame, png_out_fpath: str):
         else:
             not_processed_data.append(row['qrcode'])
     data = confusion_matrix(actual_stunting, predicted_stunting)
-    plt.rcParams.update({'font.size': 8})
+    T1, FP1, FP2, FN1, T2, FP3, FN2, FN3, T3 = data.ravel()
+    sum = T1 + FP1 + FP2 + FN1 + T2 + FP3 + FN2 + FN3 + T3
```

Comment:
This should be faster
```suggestion
    sum = sum(T1, FP1, FP2, FN1, T2, FP3, FN2, FN3, T3)
```